### PR TITLE
Adjusted reverse kinematic mecanum equation

### PR DESCRIPTION
**Pull request by computer-whisperer**
Adjusted reverse kinematic mecanum equation to take wheelbase dimensions into account.

### CODE REVIEW

File: lib/pyfrc/physics/drivetrains.py [hunk 1]
```diff
@@ -104,41 +104,41 @@ def mecanum_drivetrain(lr_motor, rr_motor, lf_motor, rf_motor, wheel_size=(8/12)
...
-    Vy = r * .25 * (lf + lr + rr + rf)
-    Vx = -r * .25 * (-lf + lr + -rr + rf)
-    
-    # For best results, k must be > 1
-    k = max(abs(Vx) + abs(Vy), 1.0)
-    
-    Vw = r * (.25/k) * (-lf + -lr + rr + rf)
-    
-    return Vx, Vy, -Vw
+    return Vx, Vy, Vw
```
Comment (virtuald) [inline lib/pyfrc/physics/drivetrains.py]: Did the sign of Vw change?

Comment (computer-whisperer) [inline lib/pyfrc/physics/drivetrains.py]: No, I inverted the signs on line 139 so that the additional negation was unnecessary.

**Comment by virtuald** [comment]
Ok, so the only thing I'm nervous about is that Ether's original paper had a bunch of math in it, and it was all dealing with wheelsize as opposed to wheelbase. You're completely disregarding the wheel size now, what's the difference?

**Comment by virtuald** [comment]
After discussion on IRC, and some testing, this seems to be ok.

**Pull request merged**